unison-src/transcripts-round-trip/reparses-with-same-hash.u

```diff
@@ -605,6 +605,8 @@ fixity = do
     (%) = Nat.mod
     ($) = (Nat.+)
     c = 1 * (2 + 3) * 4
+    minus = 1 Nat.+ 2 Nat.+ 3
+    minus2 = 1 Nat.+ (2 Nat.+ 3)
```

Comment:
Is there a reason that this is called `minus` but does addition? It is probably harmless but it seems peculiar.

Reply:
It used to be `Nat.-` :)